aicsimageio/readers/czi_reader.py

```diff
@@ -0,0 +1,740 @@
+
+        # Because dimensions outside of Y and X can be in any order and present or not
+        # we also return the dimension order string.
+        return merged  # , "".join(dims)
+
+    @staticmethod
+    def _get_coords_and_physical_px_sizes(
+        xml: ET.Element, scene_index: int, image_short_info: Dict[str, Any]
+    ) -> Tuple[Dict[str, Any], types.PhysicalPixelSizes]:
+        # Create coord dict
+        coords = {}
+
+        # Get all images
+        img_sets = xml.findall(".//Image/Dimensions/Channels")
+
+        if len(img_sets) != 0:
+            # Select the current scene
+            img = img_sets[0]
+            if scene_index < len(img_sets):
+                img = img_sets[scene_index]
+
+            # Construct channel list
+            scene_channel_list = []
+            channels = img.findall("./Channel")
+            for i, channel in enumerate(channels):
+                channel_name = channel.attrib["Name"]
+                channel_id = channel.attrib["Id"]
+                channel_ce = channel.find("./ContrastMethod")
+                channel_contrast = channel_ce.text if channel_ce is not None else ""
+                scene_channel_list.append(
+                    (f"{channel_id}" f"--{channel_name}" f"--{channel_contrast}")
+                )
+
+            # Attach channel names to coords
+            coords[DimensionNames.Channel] = scene_channel_list
+
+        # Unpack short info scales
+        list_xs = xml.findall(".//Distance[@Id='X']")
+        list_ys = xml.findall(".//Distance[@Id='Y']")
+        list_zs = xml.findall(".//Distance[@Id='Z']")
+
+        scale_xe = list_xs[0].find("./Value")
+        scale_ye = list_ys[0].find("./Value")
+        scale_ze = None if len(list_zs) == 0 else list_zs[0].find("./Value")
+        scale_x = float(str(scale_xe.text)) if scale_xe is not None else 1.0
+        scale_y = float(str(scale_ye.text)) if scale_ye is not None else 1.0
+        scale_z = float(str(scale_ze.text)) if scale_ze is not None else 1.0
+        scale_t = None
+
+        # Handle Spatial Dimensions
+        if scale_ze is not None:
+            coords[DimensionNames.SpatialZ] = np.arange(
+                0,
+                image_short_info[DimensionNames.SpatialZ][1] * scale_z,
+                scale_z,
+            )
+        if scale_y is not None:
+            coords[DimensionNames.SpatialY] = np.arange(
+                0,
+                image_short_info[DimensionNames.SpatialY][1] * scale_y,
+                scale_y,
+            )
+        if scale_x is not None:
+            coords[DimensionNames.SpatialX] = np.arange(
+                0,
+                image_short_info[DimensionNames.SpatialX][1] * scale_x,
+                scale_x,
+            )
+
+        # Time
+        if scale_t is not None:
+            coords[DimensionNames.Time] = np.arange(
+                0,
+                image_short_info["dims"].t * scale_t,
+                scale_t,
+            )
```

Comment:
I think this was directly copied from LifReader... I don't think `image_short_info` for you is a dictionary with `["dims"].t` available....

Reply:
I now see you are just setting scale_t to None up above meaning this block will never get hit.... Is this just because we don't have at timeseries file to test against??